examples/http2/index.js

```diff
@@ -0,0 +1,40 @@
+var isHttp2Supported = require('../../lib/utils').isHttp2Supported;

```

Comment:
The examples should not be reaching into Express internals.

Reply:
I think this can (and should) be exposed as a public API (isHttp2Supported)